common/src/main/scala/net/psforever/objects/serverobject/tube/SpawnTube.scala

```diff
@@ -13,22 +13,24 @@ object SpawnTube {
     new SpawnTube(tubeDef)
   }
 
-//  import akka.actor.ActorContext
```

Comment:
Did you forget to remove this commented code, or are you keeping it in for a reason?